### PR TITLE
Added a note on the order of parameters

### PR DESCRIPTION
**Pull request by ounsworth**
@wussler can you review this pr please?

### CODE REVIEW

File: draft-ounsworth-cfrg-kem-combiners.mkd [hunk 1]
```diff
@@ -373,6 +373,25 @@ The parameter fixedInfo MAY contain any of the following information:
...
 This is a non-comprehensive list, further information can be found in paragraph 5.8.2 of NIST SP800-56Ar3 {{SP800-56A}}.
 
+## Note on the order of parameters
+
+For a two-KEM instantiation, the construction is
+
+```
+KDF(counter || ct_1 || rlen(ct_i1 || ss_1 || rlen(ss_1) || 
+               ct_2 || rlen(ct_2) || ss_2 || rlen(ss_2) || 
+               k_n || fixedInfo, outputBits)
```
Comment (wussler) [inline draft-ounsworth-cfrg-kem-combiners.mkd]:
```suggestion
KDF(counter || ct_1 || rlen(ct_1) || ss_1 || rlen(ss_1) || 
               ct_2 || rlen(ct_2) || ss_2 || rlen(ss_2) || 
               fixedInfo, outputBits)
```

Comment (ounsworth) [reply draft-ounsworth-cfrg-kem-combiners.mkd]: Fixed.

Comment (wussler) [reply draft-ounsworth-cfrg-kem-combiners.mkd]: k_n is still there :sweat_smile:

Comment (ounsworth) [reply draft-ounsworth-cfrg-kem-combiners.mkd]: Oh, I missed that there were two typos in there. Got it.

**Pull request merged**